### ISSUE TITLE
"Visit the feedback forum" link doesn't have enough contrast

### ISSUE THREAD

**Issue by webinista** [department-of-veterans-affairs/vets-website#3964]
Changing `.feedback-widget:last-child` to have an `opacity` of `.6` or setting the color to `rgba(255,255,255,.6)` should fix.
